### ISSUE TITLE
“Glow for Empowerment” is not properly working on Evoker

### ISSUE THREAD

**Issue by intoache** [Hekili/hekili#3504]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

Thank you and cheers for all the efforts on this great addon! 
For my Evoker (Aug or Dev), I expect the addon to glow when it comes to the recommended empowerment stage like the option describes. But in fact it either doesn't glow on all empowerment stages or it just shines with action highlight on all stages, depending on the option "Enable Action Highlight". Is there any method to make it work properly (with either action highlight or glow), just hint me it's the right empowerment stage?

### How to Reproduce

1. Enter game as a DPS Evoker.
2. Turn on the "Glow when Empowered" Option.
3. Cast an empowerment skill.
4. See if it's glowing on the right empowerment stage.

### Snapshot (Link)

https://pastebin.com/CWQiy7Wk

### Contact Information

intoache

**Comment by intoache** [comment]
Yeah it surely has a text to show which stage i should empower to, it's just cooler and noticeable with a graphical alert

**Comment by Hekili** [comment]
I'm not able to replicate this issue immediately on target dummies.  Can you clarify a bit?

Does the addon recommend Fire Breath IV and then change to Fire Breath I while empowering and/or fail to light up at that point?  I need to understand the sequence of events for this one.  Thanks!

**Comment by intoache** [comment]
OK, the thing is, the addon recommend fire breath iv and no change, and it glows(or action highlight, i'm not sure) from phase I to  phase IV all the way, while i think it should be like no glow from I to III, and starts to glow in phase IV.

I tried to upload some photo with eternal surge, and it's all alike with other empowerment cast, hope it's clear.
![1](https://github.com/user-attachments/assets/f7992bcb-9cb9-4fb8-a86b-4cf9bfcc3dff)
![2](https://github.com/user-attachments/assets/6e522195-96fa-4fd3-b9d1-d007b24b45e8)
![3](https://github.com/user-attachments/assets/34886924-cc5c-4339-8bae-fe9596f5c388)
![4](https://github.com/user-attachments/assets/759a6926-35d9-4800-878d-8c940889758c)
![5](https://github.com/user-attachments/assets/751fe1cd-5ed3-4c6b-add7-b30ae76f4275)
![setting](https://github.com/user-attachments/assets/94e3be91-357e-464f-8f3f-51ea8f121ac8)
